src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx

```diff
@@ -101,51 +104,64 @@ class RuleConditionsForm extends React.PureComponent<Props, State> {
       <Panel>
         <PanelHeader>{t('Configure Rule Conditions')}</PanelHeader>
         <PanelBody>
+          <Feature
+            requireAll
+            features={[
+              'organizations:transaction-events',
+              'organizations:internal-catchall',
```

Comment:
Are you using this to hide these controls from alpha users while you finish off the feature?

Reply:
yup